lib/Extension/LanguageServerPhpCsFixer/Util/DiffToTextEditsConverter.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerPhpCsFixer\Util;
+
+use Phpactor\LanguageServerProtocol\Range;
+use Phpactor\LanguageServerProtocol\Position;
+use Phpactor\LanguageServerProtocol\TextEdit;
+use SebastianBergmann\Diff\Line;
+use SebastianBergmann\Diff\Parser;
```

Comment:
depending on another package here is probably OK, although in core Phpactor packages I would make this an interface and put the implementation in `Adapter/SebastianDiff` or similar.

Reply:
I've left it without interface and adapter. I'd rather expect php-cs-fixer changes to output than the changes in that `diff` package, which is one of most popular ones on packagist.org 

Unless there are other reasons for that?

Reply:
As said, it's probably OK. It's just hygienic, we shouldn't depend on random packages (regardless of how popular they are) in "core" code. Also ... is the Diff package a non-dev dependency in composer?

Reply:
it's a direct dependency of Phpactor :+1: in general though I'd rather put things that use 3rd parties behind an interface, so `Adapter/SebastianDiff/DiffToTextEditConverter` and `DiffToTextEditsConverter` is then the interface.

but :shrug: